### PR TITLE
[tests] verify the RA/RS fields in border routing counters

### PR DESCRIPTION
**Pull request by superwhd**

### CODE REVIEW

File: tests/scripts/thread-cert/node.py [hunk 1]
```diff
@@ -469,7 +475,9 @@ def read_border_routing_counters_delta(self):
                     assert (key in new_counters)
                     value = [new_counters[key][0] - old_counters[key][0], new_counters[key][1] - old_counters[key][1]]
                     delta_counters[key] = value
-        delta_counters = {key: value for key, value in delta_counters.items() if value[0] and value[1]}
+        delta_counters = {
+            key: value for key, value in delta_counters.items() if not isinstance(value, int) and value[0] and value[1]
```
Comment (jinran-google) [inline tests/scripts/thread-cert/node.py:479]: Without `isinstance`, it should raise an error when old counters don't exist. Why didn't the old implementation cause an error?

**Review by jinran-google — APPROVED** [review]
LGTM overall. A question below:

File: tests/scripts/thread-cert/node.py (hunk 1 above)
Comment (superwhd) [reply tests/scripts/thread-cert/node.py:479]: In the old implementation, all the values were `(int, int)` so it wouldn't cause an error.

Comment (jinran-google) [reply tests/scripts/thread-cert/node.py:479]: Makes sense. But this makes `read_border_routing_counters_delta` return only `inbound_unicast`, `inbound_multicast`, `outbound_unicast`, `outbound_multicast`. Is this expected?

Comment (superwhd) [reply tests/scripts/thread-cert/node.py:479]: This is expected. For now we don't want to check the delta of RA/RS in a specific period.

**Review by jwhui — APPROVED** [review]

**Pull request merged**